pkg/occlient/occlient.go

```diff
@@ -1046,8 +1047,35 @@ func (c *Client) FollowBuildLog(buildName string) error {
 	stdout := color.Output
 
 	if _, err = io.Copy(stdout, rd); err != nil {
-		return errors.Wrapf(err, "error streaming logs for %s", buildName)
+		return errors.Wrapf(err, "error streaming logs for %s", name)
+	}
+
+	return nil
+}
+
+// Display DeploymentConfig log to stdout
+func (c *Client) DisplayDeploymentConfigLog(buildName string) error {
```

Comment:
the argument is called `buildName`  but this function accepts DeploymentConfig name as an argument